File: blink.c
#include <wiringPi.h>
int main (void)
{
  #define PIN 7
  wiringPiSetup () ;
  pinMode(PIN, OUTPUT) ;
  for (;;)
  {
    digitalWrite (PIN,  LOW) ; delay (500) ;
    digitalWrite (PIN, HIGH) ; delay (125) ;
  }
  return 0 ;
}

File: blink.py
import RPi.GPIO as GPIO
import time
PIN = 7
LOW = 0
HIGH = 1

def clockIn():
	GPIO.output(PIN,0)
	time.sleep(0.5)
	GPIO.output(PIN,1)
	time.sleep(0.125)

GPIO.setwarnings(False)
GPIO.setmode(GPIO.BOARD)
GPIO.setup(PIN, GPIO.OUT)
for i in range(1,20):
	clockIn()
GPIO.cleanup()

